app/views/finance/lead_providers/show.html.erb

```diff
@@ -0,0 +1,98 @@
+<% content_for :title, "Lead Provider Payment Breakdown" %>
+<%
+  def band_index_to_heading(number)
+    ("A".."Z").to_a[number]
+  end
+%>
```

Comment:
This should be a helper method in the controller.

Reply:
This would be best placed as a decorator.

Reply:
@blackrat is going to rustle up a patch :+1: